port/win/port_win.cc

```diff
@@ -142,7 +142,7 @@ DIR* opendir(const char* name) {
     return nullptr;
   }
 
-  strncpy_s(dir->entry_.d_name, dir->data_.name, strlen(dir->data_.name));
+  strcpy_s(dir->entry_.d_name, sizeof(dir->entry_.d_name), dir->data_.name);
```

Comment:
Again, this will not compile with MinGW 6.2.0. Why should we support only >=6.3.0 just for a negligible style improvement?